api/internal/handlers/kubernetes/install.go

```diff
@@ -77,6 +79,79 @@ func (h *Handler) GetInstallationStatus(w http.ResponseWriter, r *http.Request)
 	utils.JSON(w, r, http.StatusOK, status, h.logger)
 }
 
+// PostRunAppPreflights handler to run install app preflight checks
+//
+//	@ID				postKubernetesInstallRunAppPreflights
+//	@Summary		Run install app preflight checks
+//	@Description	Run install app preflight checks using current app configuration
+//	@Tags			kubernetes-install
+//	@Security		bearerauth
+//	@Produce		json
+//	@Success		200	{object}	types.InstallAppPreflightsStatusResponse
+//	@Router			/kubernetes/install/app-preflights/run [post]
+func (h *Handler) PostRunAppPreflights(w http.ResponseWriter, r *http.Request) {
+	preflightBinary, err := h.cfg.KubernetesConfig.Installation.PathToEmbeddedBinary("kubectl-preflight")
```

Comment:
per our discussion with joao in slack, isnt the recommendation here to use a different interface rather than the installation?